### ISSUE TITLE
Create UpgradeHook for automatic minor update admin-ack

### ISSUE THREAD

**Issue by bastjan** [appuio/component-openshift-upgrade-controller#51]
## Context

OpenShift requires a manual admin ack when upgrading minor versions. See [Providing the administrator acknowledgment](https://docs.openshift.com/container-platform/4.14/updating/preparing_for_updates/updating-cluster-prepare.html#update-preparing-ack_updating-cluster-prepare).

If we set a release channel in project syn we implicitly acknowledge that a minor upgrade can be done.

We have had cases where we forgot to do the admin ack after a channel upgrade and this did lead to mainenance/on-call alerts.

We should automate the admin ack.

## Alternatives

Do manual work and get on-call alerts.

**Comment by bastjan** [comment]
Ack CM from lpg-2

```
apiVersion: v1
data:
  ack-4.11-kube-1.25-api-removals-in-4.12: "true"
  ack-4.12-kube-1.26-api-removals-in-4.13: "true"
  ack-4.13-kube-1.27-api-removals-in-4.14: "true"
kind: ConfigMap
metadata:
  annotations:
    include.release.openshift.io/ibm-cloud-managed: "true"
    include.release.openshift.io/self-managed-high-availability: "true"
    include.release.openshift.io/single-node-developer: "true"
    release.openshift.io/create-only: "true"
  name: admin-acks
  namespace: openshift-config
```

**Comment by bastjan** [comment]
Sample cluster version object with {{AdminAckRequired}} condition.

```yaml
apiVersion: config.openshift.io/v1
kind: ClusterVersion
metadata:
  name: version
spec:
  channel: stable-4.13
status:
  availableUpdates:
    - channels:
        - ...
        - stable-4.13
        - stable-4.14
  conditions:
    - lastTransitionTime: '2023-09-19T20:58:06Z'
      message: >-
        Kubernetes 1.27 and therefore OpenShift 4.14 remove several APIs which
        require admin consideration. Please see the knowledge article
        https://access.redhat.com/articles/6958395 for details and instructions.
      reason: AdminAckRequired
      status: 'False'
      type: Upgradeable
```

**Comment by bastjan** [comment]
Fixed by #56
